config/jobs/kubernetes/sig-cloud-provider/gcp/gpu/gpu-gce.yaml

```diff
@@ -5,7 +5,7 @@ presets:
   # Note: The GCE Node image used may have a dependency on the nvidia-driver-installer image defined in https://github.com/kubernetes/kubernetes/blob/master/test/e2e/testing-manifests/scheduling/nvidia-driver-installer.yaml
   # If updating the image defined here, the cos-gpu-installer image may need to updated to support the corresponding COS image.
   - name: KUBE_GCE_NODE_IMAGE
-    value: cos-85-13310-1041-9
+    value: cos-97-16919-103-16
   - name: KUBE_GCE_NODE_PROJECT
     value: cos-cloud
   - name: NODE_ACCELERATORS
```

Comment:
We also need to update the cos-gpu-installer tag. We should update it to `v2.0.27` since it's the version used in COS 97 

![image](https://user-images.githubusercontent.com/486583/200044718-292c2aa3-1604-4096-bd47-2ebe2c50568f.png)


Can you also please update https://github.com/kubernetes/kubernetes/blob/master/test/e2e/testing-manifests/scheduling/nvidia-driver-installer.yaml#L56 to `gcr.io/cos-cloud/cos-gpu-installer:v2.0.27`?

Thanks!

Reply:
filed https://github.com/kubernetes/kubernetes/pull/113632